<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>José Manuel Díez Botella</author>
 <class>subcuentas</class>
 <widget class="QDialog" name="subcuentas">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>464</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SUBCUENTAS</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="treeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Subcuenta:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="codigolineEdit">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Descripción:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="descripcionlineEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="datosgroupBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Datos accesorios</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QGridLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="2" column="0">
             <widget class="QLabel" name="TextLabel1_2">
              <property name="text">
               <string>Domicilio:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="3">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Nombre Comercial:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="TextLabel2">
              <property name="text">
               <string>Razón social:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="1" colspan="3">
             <widget class="QLineEdit" name="DomicilioLineEdit">
              <property name="maxLength">
               <number>80</number>
              </property>
             </widget>
            </item>
            <item row="0" column="2" colspan="2">
             <widget class="QLineEdit" name="NombreLineEdit">
              <property name="maxLength">
               <number>80</number>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="ncomerciallineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="TextLabel2_2">
              <property name="text">
               <string>Población:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="PoblacionLineEdit">
              <property name="maxLength">
               <number>80</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="TextLabel3">
              <property name="text">
               <string>Provincia:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ProvinciaLineEdit">
              <property name="maxLength">
               <number>80</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="TextLabel4">
              <property name="text">
               <string>Código Postal:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="CpLineEdit">
              <property name="maxLength">
               <number>40</number>
              </property>
             </widget>
            </item>
            <item row="0" column="9">
             <widget class="QLineEdit" name="NifComLineEdit">
              <property name="maxLength">
               <number>40</number>
              </property>
             </widget>
            </item>
            <item row="1" column="7">
             <widget class="QLabel" name="TextLabel9">
              <property name="text">
               <string>Fax:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QLineEdit" name="NifLineEdit">
              <property name="maxLength">
               <number>40</number>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="TextLabel8">
              <property name="text">
               <string>Tfno:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="TextLabel7">
              <property name="text">
               <string>País:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="4" colspan="3">
             <widget class="QLineEdit" name="TfnoLineEdit">
              <property name="maxLength">
               <number>80</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QLineEdit" name="PaisLineEdit">
              <property name="maxLength">
               <number>80</number>
              </property>
             </widget>
            </item>
            <item row="1" column="8" colspan="2">
             <widget class="QLineEdit" name="FaxLineEdit">
              <property name="maxLength">
               <number>80</number>
              </property>
             </widget>
            </item>
            <item row="0" column="3" colspan="2">
             <widget class="QLabel" name="TextLabel5">
              <property name="text">
               <string>NIF/CIF:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="6" colspan="3">
             <widget class="QLabel" name="TextLabel6">
              <property name="text">
               <string>NIF Comunitario:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="TextLabel10">
              <property name="text">
               <string>Email:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="ObservacionesLineEdit">
              <property name="maxLength">
               <number>80</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QLabel" name="TextLabel11">
              <property name="text">
               <string>Observaciones:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="2">
             <widget class="QLineEdit" name="EmailLineEdit">
              <property name="maxLength">
               <number>80</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Cuenta domiciliación (CCC):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ccclineEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Cuota:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="cuotalineEdit">
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="borrarpushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&amp;Borrar</string>
           </property>
           <property name="icon">
            <iconset resource="libroK.qrc">
             <normaloff>:/images/librok-17.png</normaloff>:/images/librok-17.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="aceptarpushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Aceptar</string>
           </property>
           <property name="icon">
            <iconset resource="libroK.qrc">
             <normaloff>:/images/librok-42.png</normaloff>:/images/librok-42.png</iconset>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="salirpushButton">
           <property name="text">
            <string>&amp;Salir</string>
           </property>
           <property name="icon">
            <iconset resource="libroK.qrc">
             <normaloff>:/images/librok-52.png</normaloff>:/images/librok-52.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>codigolineEdit</tabstop>
  <tabstop>descripcionlineEdit</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>NombreLineEdit</tabstop>
  <tabstop>ncomerciallineEdit</tabstop>
  <tabstop>DomicilioLineEdit</tabstop>
  <tabstop>PoblacionLineEdit</tabstop>
  <tabstop>ProvinciaLineEdit</tabstop>
  <tabstop>CpLineEdit</tabstop>
  <tabstop>NifLineEdit</tabstop>
  <tabstop>NifComLineEdit</tabstop>
  <tabstop>PaisLineEdit</tabstop>
  <tabstop>TfnoLineEdit</tabstop>
  <tabstop>FaxLineEdit</tabstop>
  <tabstop>EmailLineEdit</tabstop>
  <tabstop>ObservacionesLineEdit</tabstop>
  <tabstop>borrarpushButton</tabstop>
  <tabstop>aceptarpushButton</tabstop>
  <tabstop>salirpushButton</tabstop>
 </tabstops>
 <resources>
  <include location="libroK.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>salirpushButton</sender>
   <signal>clicked()</signal>
   <receiver>subcuentas</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>632</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>609</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
